python/freetensor/core/remote_task_scheduler.py

```diff
@@ -0,0 +1,735 @@
+from __future__ import annotations
```

Comment:
Annotations seem to be supported since Python 3.5, but we only support Python >= 3.8, so there is no need to import from `__future__`.

Reply:
Oops, I will fix this in my next commit

Reply:
Since it raises warning on my server when I remove this line, this section will remain consistent for the time being

Reply:
What version of Python are you using? We can't even parse the AST correctly for Python <3.8.

Reply:
I believe that my python version is OK.(python 3.10.4) I think it's a matter of IDE. I rely heavily on the IDE's code checking

Reply:
`__future__.annotations` additionally enables the usage of yet unfinished classes in annotations, which is a Python 4.0 planned feature. It is also used in `staging.py` and should be ok.